### PR TITLE
Document production setup

### LINKED ISSUE #6: Add deployment docs
Should include:
- Sample config files.
- Supported platforms.
- Recommended production settings.
- Maintenance, backup, and restore instructions.

### PR DESCRIPTION
**Pull request by pg9182**
Doesn't match the current setup exactly since I've removed references to internal stuff, but it is complete and generally accurate.

closes #6

### CODE REVIEW

File: docs/PRODUCTION.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    ```bash
+    # option 1: latest version
+    sudo GOBIN=/usr/local/bin go install -v -trimpath github.com/r2northstar/atlas/cmd/...@main
+    ```
+
+    ```bash
+    # option 2: from local git clone
+    git clone https://github.com/r2northstar/atlas
+    cd atlas
+    sudo GOBIN=/usr/local/bin go install -v -trimpath github.com/r2northstar/atlas/cmd/...
```
Comment (GeckoEidechse) [inline docs/PRODUCTION.md:29]: out of curiosity, what's with the triple `...`? ^^

**Review by GeckoEidechse — APPROVED** [review]
Very nicely formatted :eyes: 

Cannot say much about the content but I think I should be able to follow it if need be :P

I assume dev docs will follow soon? ^^

File: docs/PRODUCTION.md (hunk 1 above)
Comment (pg9182) [reply docs/PRODUCTION.md:29]: Think of it as a `**` glob, but a Go thing and for matching folders (i.e., packages) only.

**Comment by pg9182** [comment]
> Cannot say much about the content but I think I should be able to follow it if need be :P

Yes, and I'll document some of the other aspects of the setup in the internal repo.

> I assume dev docs will follow soon? ^^

Yes, then contributing ones.

**Pull request merged**